applications/conference/src/conf_discovery_req.erl

```diff
@@ -436,3 +440,27 @@ create_conference(JObj, Digits, Call) ->
         %%   or they joined by the discovery event having the conference id
         _Else -> Conference
     end.
+
+-spec get_pin_timeout(kapps_conference:conference()) -> pos_integer().
+get_pin_timeout(Conference) ->
+    AccountId = kapps_conference:account_id(Conference),
+    JObj = kapps_conference:conference_doc(Conference),
+    lager:debug("pin timeout request account:~p conference:~p", [AccountId, kapps_conference:id(JObj)]),
+    kz_json:get_value(<<"pin_timeout">>, JObj, get_account_pin_timeout(AccountId)).
+
+-spec get_account_pin_timeout(ne_binary()) -> pos_integer().
+get_account_pin_timeout(AccountId) ->
+    kapps_account_config:get(AccountId, ?CONFIG_CAT, <<"pin_timeout">>, get_default_pin_timeout()).
```

Comment:
use `kapps_account_config:get_global` so that reseller can set defaults to their accounts